bot/exts/utilities/bookmark.py

```diff
@@ -73,7 +72,12 @@ def __init__(self, bot: Bot):
             name="Bookmark",
             callback=self._bookmark_context_menu_callback,
         )
+        self.delete_book_mark_context_menu = discord.app_commands.ContextMenu(
+            name="Delete bookmark",
+            callback=self._delete_bookmark_context_menu_callback,
+        )
```

Comment:
The docs say:
> Setting an empty permissions field will disallow anyone except server administrators from using the command in a guild.

By default, upon instantiation, it defaults to none since the callback doesn't have the `__discord_app_commands_default_permissions__` attribute, so I'm wondering if that counts as the same ?

If not, i'll just add this then
```py
self.delete_book_mark_context_menu.default_permissions = []
```